main.go

```diff
@@ -210,13 +220,10 @@ func avsSyncMain(cliCtx *cli.Context) error {
 	}
 	logger.Infof("Sleeping for %v before first sync, so that it happens at %v", sleepBeforeFirstSyncDuration, time.Now().Add(sleepBeforeFirstSyncDuration))
 
-	// Create new prometheus registry
-	reg := prometheus.NewRegistry()
-
 	avsSync := avssync.NewAvsSync(
 		logger,
-		avsReader,
-		avsWriter,
+		*avsReader,
+		*avsWriter,
```

Comment:
is this intended? typically you'd want to pass a pointer around, but haven't looked at the code (for eg are the methods on avsReader defined on pointers or structs)?

Reply:
Sorry holdover from when these were taking interfaces, will change!

Reply:
https://github.com/Layr-Labs/avs-sync/pull/45/commits/2ac06af8b5edfd89342006e0687a6cb97447162d